test/unit-test/scripts/npm/prepublish/lib/build-spec.js

```diff
@@ -1,37 +1,31 @@
-const sinon = require( 'sinon' );
-const chai = require( 'chai' );
-const expect = chai.expect;
-const sinonChai = require( 'sinon-chai' );
 const childProcess = require( 'child-process-promise' );
 const path = require( 'path' );
 const rootPath = require( '../root-path' );
 const libPath = path.join( rootPath, 'scripts', 'npm', 'prepublish', 'lib' );
 const util = require( libPath );
 let execStub;
 
-chai.use( sinonChai );
-
 describe( 'build', () => {
   beforeEach( () => {
-    execStub = sinon.stub( childProcess, 'exec' );
+    execStub = jest.spyOn( childProcess, 'exec' );
   } );
 
   afterEach( () => {
-    execStub.restore();
+    execStub.mockRestore();
   } );
 
-  it( 'should trigger a build of all CF components', () => {
+  xit( 'should trigger a build of all CF components', () => {
```

Comment:
Does this need to be silenced? If so do we know why and could you add a comment if so?

Reply:
Hm, nope! Fixed